aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -106,17 +106,17 @@ compute_result_type(at::ArrayRef<OperandInfo> operands,
   return compute_result_type(operands, predicates...);
 }
 
-std::tuple<Device, ScalarType> TensorIterator::compute_common_type() {
+std::tuple<Device, ScalarType> compute_common_type_(at::ArrayRef<OperandInfo> operands) {
```

Comment:
Should be `static`, as far as I can tell this function is a helper function used only in this file.

Reply:
Also, the naming is a little strange: why does it end with an underscore? I think it should be sufficient to name it `compute_common_type`

Reply:
Changed to static